cache/edge_lambdas/toggler.js

```diff
@@ -10,7 +10,15 @@
 // }
 
 // This is mutable for testing
-let tests = [];
+let tests = [
+  {
+    id: 'showShortTitles',
+    title: 'Short exhibition titles',
+    shouldRun: request => {
+      return !request.uri.match(/^\/works\/*/);
```

Comment:
Makes perfect sense, "Should run everywhere but works" - like you said.

Reply:
Although saying that, looking at the issue outlining the tests, we're only measure `/whats-on` and `/`?

Reply:
This would us avoid capturing people who don't hit other pages (from Google etc).

Reply:
Don't think I follow?

Reply:
Sticking to `/whats-on` and `/` makes sense given that's what we should be testing though.

Reply:
Would `request.uri.match(/^\/whats-on.*/) || request.uri === '/';` make sense?

Reply:
Doesn't `.*` in the what's on regex mean 1 or more characters?